packages/terra-html-table/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 Changelog
 =========
+### Changed
+* Update docs to use mdx syntax.
```

Comment:
whoops, this should be under the unreleased section

Reply:
done here https://github.com/cerner/terra-core/pull/2850/commits/5db8bce68a204971a17a3039ecf58b9849ca0e1e.